src/main/java/org/openstreetmap/atlas/geography/atlas/change/FeatureChange.java

```diff
@@ -522,6 +519,27 @@ public void save(final WritableResource resource)
         new FeatureChangeGeoJsonSerializer(true).accept(this, resource);
     }
 
+    /**
+     * Add a new key value pair to this FeatureChange's meta-data
+     * 
+     * @param key
+     *            The key
+     * @param value
+     *            The value
+     */
+    public void setMetaData(final String key, final String value)
```

Comment:
Maybe call this `addMetaData` instead. I think `set` implies that it will wipe the current metadata which seems not to be the case.